tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,6 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    #raise Exception("Complete this test according to comments below.")
+    assert result == False
```

Comment:
Rad! We can also use `assert not result` and that does the same thing as what you wrote here!